README.md

```diff
@@ -780,6 +780,31 @@ pageSize = 100
 res = mds.readCurveRange(100042422, page, pageSize, versionFrom="2016-12-20" , versionTo="2019-03-12")
 ```
 
+### Search the MarketData collection with faceted results
+
+Using MarketDataService is possible to query and search the MarketData collection with faceted results. Supports paging, filtering and free text.
+
+```Python
+from Artesian.MarketData import MarketDataService
+
+mds = MarketDataService(cfg)
+
+```
+
+To list MarketData curves
+
+```Python
+page = 1
+pageSize = 100
+searchText = "Riconsegnato_"
+filters = [
```

Comment:
nope. I do remember that correct json encoding was supported for dictionaries without having to express them.

check the _Client class and serialization helpers.

Reply:
https://github.com/ARKlab/Artesian.SDK-Python/blob/master/src/Artesian/_ClientsExecutor/ArtesianJsonSerializer.py

add another dedicated deser pair for the new Dict